libraries/AP_GPS/AP_GPS_NOVA.h

```diff
@@ -114,26 +114,26 @@ class AP_GPS_NOVA : public AP_GPS_Backend
 
     struct PACKED bestpos
     {
-        uint32_t solstat;
-        uint32_t postype;
-        double lat;
-        double lng;
-        double hgt;
-        float undulation;
-        uint32_t datumid;
-        float latsdev;
-        float lngsdev;
-        float hgtsdev;
+        uint32_t solstat;      //!< Solution status
```

Comment:
I have to ask whats the meaning you use in comments? I see things that are `//` `///<` and `//!<` and it just looks really inconsistent.